backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java

```diff
@@ -68,4 +69,8 @@ public CompletableFuture<DeviceType> getDeviceType(
       ApiTestOrder apiTestOrder, DataFetchingEnvironment dfe) {
     return _testOrderDeviceTypeDataLoader.load(apiTestOrder.getWrapped(), dfe);
   }
+
+  public TestResult getResult(ApiTestOrder apiTestOrder, DataFetchingEnvironment dfe) {
+    return apiTestOrder.getWrapped().getResult();
+  }
```

Comment:
@zdeveloper this is how I eventually got around that frontend persistence bug I mentioned - this seems to force GraphQL to load the latest Result from the database, instead of caching it.